libr/core/cmd_debug.c

```diff
@@ -5037,6 +5037,10 @@ static int cmd_debug(void *data, const char *input) {
 			r_core_cmd_help (core, help_msg_dd);
 			break;
 		}
+		if (core->dbg->pid == -1) {
```

Comment:
usually we check if core->config have cfg.debug set to true, and there's also an api tlo chk if the debugger child is dead or not